src/utils/formatters.js

```diff
@@ -454,11 +299,6 @@ export function formatCents(cents, minimumFractionDigits = 2, maximumFractionDig
   return (cents / 100).toLocaleString(undefined, { minimumFractionDigits, maximumFractionDigits });
```

Comment:
The difference between this `formatCents` and the one inside `shared/formatters` is that the one inside of `ahared/formatters` has the following code above this line. I am not certain if I should put that in this function or not.

> if (!isFinite(cents)) {
    return '';
  }

Reply:
Internally this is doing https://github.com/lodash/lodash/blob/4.17.15/lodash.js#L11626. This effectively makes the function return an empty string in the instance this function is called without an appropriate input. 

When we don't have this, we likely get `NaN` when we have an invalid input. 

I don't think we need to include it. ~~That said, from a consistency standpoint it might be better to add the following to the the `formatCents` function.~~ (I see an instance where we are returning null instead of an empty string when it fails this check.)
```suggestion
  if (!Number.isFinite(cents)) {
    return '';
  }
  return (cents / 100).toLocaleString(undefined, { minimumFractionDigits, maximumFractionDigits });
```

Reply:
If time permits I'd advocate for looking into why that one function (`convertFromThousandthInchToInch`) returns null and see if we can modify this without breakages and adding the suggestion above.

In general it looks like most of these function (those that are using the Number.isFinite) are called on data from the DB so they probably never go into this if scope.

Reply:
I've created a new issue based out of this discussion.
[https://dp3.atlassian.net/browse/MB-12306](https://dp3.atlassian.net/browse/MB-12306)